android/app/src/main/java/app/covidshield/receiver/worker/ExposureCheckSchedulerWorker.kt

```diff
@@ -32,9 +32,10 @@ class ExposureCheckSchedulerWorker (val context: Context, parameters: WorkerPara
         try {
             val enIsEnabled = exposureNotificationClient.isEnabled.await()
             val enStatus = exposureNotificationClient.status.await()
-            if (!enIsEnabled || enStatus.contains(ExposureNotificationStatus.INACTIVATED)){
+
+            if (!enIsEnabled || enStatus.contains(ExposureNotificationStatus.INACTIVATED)) {
                 Log.d("background", "ExposureCheckSchedulerWorker - ExposureNotification: Not enabled or not activated")
-                MetricsService.publishDebugMetric(2.1, context, "ExposureNotification: Not enabled or not activated.");
+                MetricsService.publishDebugMetric(2.1, context, "ExposureNotification: enIsEnabled = $enIsEnabled AND enStatus = ${enStatus.firstOrNull()?.ordinal}.");
```

Comment:
Shouldn't `$enIsEnabled` be contained in parenthesis?

Reply:
It works without it just because we are including a basic variable. As soon as it starts getting more complex you do not have a choice but to use `#{}`. Android Studio gives me a warning if I use `#{enIsEnabled}` (which is also valid)